rxandroidble/src/test/groovy/com/polidea/rxandroidble/RxBleClientTest.groovy

```diff
@@ -79,6 +79,49 @@ class RxBleClientTest extends Specification {
         1 * bleAdapterWrapperSpy.startLeScan(_) >> true
     }
 
+    def "should start BLE scan if bluetooth is enabled before subscribing to the scan observable"() {
```

Comment:
This description does not seem to match the test. Here it is checking if `bleAdapterWrapperSpy.isBluetoothEnabled()` was not called in line 92